### ISSUE TITLE
Service assigned to only a hostgroup or servicegroup does not render the commandline

### ISSUE THREAD

**Issue by jandrunas** [opinkerfi/adagios#511]
If you create a service and don't assign it to any hosts, and only a host group or service group, when you go to the Monitoring tab for the service it doesn't render the  Command Line or Expanded Command Line boxes.

Looking at the javascript console I see

jquery-1.9.1.min.js:5 sendjquery-1.9.1.min.js:5 b.extend.ajaxpynag.js:340 adagios.rest.pynag.check_commandadagios_status.js:163 self.generate_input_fields-8607336800809462953:3326 (anonymous function)jquery-1.9.1.min.js:3 cjquery-1.9.1.min.js:3 p.fireWithjquery-1.9.1.min.js:3 b.extend.readyjquery-1.9.1.min.js:3 H

 error: [object Object]
adagios_status.js:334self.error
adagios_status.js:243(anonymous function)
jquery-1.9.1.min.js:3c
jquery-1.9.1.min.js:3p.fireWith
jquery-1.9.1.min.js:5k
jquery-1.9.1.min.js:5r

**Comment by ratchetnclank** [comment]
Yes this issues exists on the current release branch which you are on.

**Comment by mortenf** [comment]
This issue is still a problem.

A work around is adding a host when creating the service, save, then set check command, remove host and save again.
